plutus-playground/plutus-playground-server/usecases/CrowdFunding.hs

```diff
@@ -68,23 +64,42 @@ contributionScript cmp  = ValidatorScript val where
                 infixr 3 &&
                 (&&) :: Bool -> Bool -> Bool
                 (&&) = $$(PlutusTx.and)
+                -- In Haskell we can define new operators. We import
```

Comment:
Usually people put comment blocks *above* the code they're describing, no?

Reply:
true.. I moved all of the comments a few lines up.